features/step_definitions/location_directory_steps.rb

```diff
@@ -102,6 +102,6 @@
   expect(Location.current.first).not_to be_hidden
 end
 
-Then(/^a new location version has been created$/) do
+Then(/^I see that the location has a newer version$/) do
   expect(Location.count).to eq(2)
```

Comment:
Sorry just missed this, but is this actually asserting there's a newer version? Should it be checking the version numbers?